subtree.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
    * @param {function} nodeCallback (optional) A callback that receives the node-like and the created element after attributes are copied but before the subtree is created.
    * @returns {JSDOM.Node} The DOM subtree
    */
-  constructSubtreeForNode: function(document, nodeLike, nodeCallback) {
+  constructSubtreeForNode: function(document, nodeLike, iframe, nodeCallback) {
```

Comment:
See my comment on parameter names for boolean parameters in serialize.js - this should probably match

Reply:
fixed in newest commit